components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -280,7 +288,7 @@ export class GithubApp {
         updatePrPromise.catch(err => log.error(err, "Error while updating PR body", { contextURL }));
     }
 
-    protected onPrAddLabel(config: WorkspaceConfig | undefined, user: User, ctx: Context, prebuildStartPromise: Promise<StartPrebuildResult> | undefined) {
+    protected onPrAddLabel(config: WorkspaceConfig | undefined, ctx: Context, prebuildStartPromise: Promise<StartPrebuildResult> | undefined) {
```

Comment:
The `addLabel` feature doesn't work anymore and should likely be removed: https://github.com/gitpod-io/gitpod/issues/3303 (from the code and the docs).

Reply:
Awesome! Will tackle this.

Reply:
done